buildSrc/src/main/kotlin/test/server/tests/Auth.kt

```diff
@@ -129,7 +129,8 @@ internal fun Application.authTestServer() {
                     val token = call.request.headers["Authorization"]
                     if (token.isNullOrEmpty() || token.contains("invalid")) {
                         call.response.header(HttpHeaders.WWWAuthenticate, "Bearer realm=\"TestServer\"")
-                        call.respond(HttpStatusCode.Unauthorized)
+                        val status = call.request.queryParameters["status"]?.toIntOrNull() ?: 401
```

Comment:
Nit:
```suggestion
                        val status = call.request.queryParameters["status"]?.toIntOrNull() ?: HttpStatusCode.Unauthorized.value
```

Reply:
I thought that the value here should be equal to the value used in the default `isUnauthorized` implementation.
It is a very minor comment, you can ignore it :)